### PR TITLE
feat: widget terms and conditions

### PR DESCRIPTION
**Pull request by shoom3301**
Source: https://docs.google.com/document/d/11MFz0nMYCtv3ztOgXh9lMxfIk2PAhfCYxo3vasMiReI/edit
Context: https://cowservices.slack.com/archives/C05QVFJUX9R/p1710942507985319

I've used https://www.gdoctohtml.com/ to convert google doc to HTML

### CODE REVIEW

**Review by elena-zh — COMMENTED** [review]
Hey @shoom3301 , great!

Some nitpicks: 
1. The page is hard to find, Maybe add a button like Ordan suggests? 
image
2. Proposal from my side is to add a link to these terms page into the footer 
image
3. There should be a period in the end 
period
4. Looks like space is missing here 
space
5. It would be great to add a mailto link here
mailto

Thanks!

File: pages/index.tsx [hunk 1]
```diff
@@ -28,7 +28,17 @@ export default function HomePage({ metricsData, siteConfigData }: HomeProps) {
 
 export const getStaticProps: GetStaticProps<HomeProps> = async () => {
   const siteConfigData = CONFIG
-  const { volumeUsd, volumeEth } = await cowSdk.cowSubgraphApi.getTotals()
+  let volumeUsd = 0
+  let volumeEth = 0
+
+  // Don't fail when couldn't get Subgraph data
```
Comment (alfetopito) [inline pages/index.tsx:34]: Shouldn't this be an isolated PR in itself?
Then we can release it independently from the T&C

Comment (shoom3301) [reply pages/index.tsx:34]: https://github.com/cowprotocol/cow-fi/pull/170

**Comment by shoom3301** [comment]
@elena-zh thank you!

1. There is a link in "Important Disclaimer" message. Have Odran mentioned another place we should put the link? Sorry, probably I missed that part
2. Makes sense, added it
3. Fixed
4. Fixed
5. Fixed

image

**Comment by elena-zh** [comment]
Hey @shoom3301 , 
1. yes, Ordan mentioned about it in https://cowservices.slack.com/archives/C05QVFJUX9R/p1710942507985319
2. case 5 is not fixed :( 
image
3. And could you start Terms and Conditions words in the footer with capitals? 
image

Thanks!

**Review by elena-zh — APPROVED** [review]
Great, thank you!

**Review by alfetopito — APPROVED** [review]

**Comment by elena-zh** [comment]
Hey @alfetopito , paragraphs order  is mixed up here
image

Could you please fix this?

**Comment by alfetopito** [comment]
> Hey @alfetopito , paragraphs order is mixed up here image
> 
> Could you please fix this?

Oops, git commit mess up. Fixed

**Comment by elena-zh** [comment]
Looks good to me now!

**Pull request merged**